src/pages/sighting/SightingCore.jsx

```diff
@@ -119,18 +132,29 @@ export default function SightingCore({
       /> */}
       <ConfirmDelete
         open={deleteDialogOpen}
-        onClose={() => setDeleteDialogOpen(false)}
+        onClose={() => {
```

Comment:
The error should probably be cleared here too. Currently if the dialog is closed and re-opened, the message reverts to the sighting deletion confirmation, but the error alert is still visible.

Reply:
Nice catch.

Reply:
Good point.